modules/virt-listing-vmis-web.adoc

```diff
@@ -0,0 +1,21 @@
+
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-manage-vmis-web.adoc
+
+[id="virt-listing-vmis-web_{context}"]
+= Listing standalone virtual machine instances using the web console
+
+Using the web console, you can list and view standalone virtual machine instances (VMIs) in your cluster that are not owned by virtual machines (VMs).
+
+[NOTE]
+====
```

Comment:
I would shorten this note. Maybe something like "To list all VMIs in your cluster, use the CLI. VMIs that are owned by VMs or other objects are not shown in the web console."